src/components/Community/Contribute/index.tsx

```diff
@@ -94,11 +94,11 @@ const Contribute: React.FC<{ theme: ICommunitySectionTheme }> = ({ theme }) => {
                 <Link
                   className={sharedStyles.button}
                   style={theme}
-                  href="mailto:info@dvc.org?subject=I want to become an Ambassador!"
+                  href="https://dvc.org/blog/dvc-ambassador-program-announcement"
                   target="_blank"
```

Comment:
@andronovhopf @rogermparent we can remove `target=_blank` in this case?

Reply:
I'd say so. Our Link wrapper will actually add `target="blank"` because it detects this as an absolute link. If we want this link not to behave as `target="_blank"`, I'd suggest changing it to root-relative (i.e. `/blog/dvc-ambassador-program-announcement`) on top of removing `target="blank"` so the wrapper detects it as relative. Using root-relative for inter-site links is best practice for us as well.

Reply:
@rogermparent sounds good! would mind making that change please?